cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/PythonLanguageFrontend.kt

```diff
@@ -105,6 +105,13 @@ class PythonLanguageFrontend(config: TranslationConfiguration, scopeManager: Sco
 
             // run python function parse_code()
             tu = interp.invoke("parse_code", code, path, this) as TranslationUnitDeclaration
+
+            if (config.matchCommentsToNodes) {
+                // Parse comments and attach to nodes
+                val pyOptional = javaClass.getResource("/CPGPython/_comment_parsing.py")
+                interp.exec(pyOptional?.readText())
```

Comment:
shouldn't the whole exec be wrap in a `?.let` or `if != null`, otherwise, you will execute an empty string

Reply:
I made this conditional, the resource should not be missing, however, now it is safer. If the resource is missing comments are not extracted nor matched